### PR TITLE
Jefferson Alvarez Alvarez's Homework

### PR DESCRIPTION
**Pull request by jefferal1995**

### CODE REVIEW

File: bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    </head>
+    <body>
+        <article class="main-container">
+            <section class="magazine-section">
+                <h1 class="title">CLARA</h1>
+            </section>
+            <section class="magazine-section">
+                <aside class="block-50">
+                    <section class="text-section">
+                        <p class="paragraph-large">¡FELIZ</p>
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html]: Unnecessary used of Capital letter. Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/

File: bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                        <p class="paragraph-large">PELO</p>
+                        <p class="paragraph-large serif">NUEVO!</p>
+                        <p class="paragraph-medium">CORTES, COLORES</p>
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html]: Seems info in the same context therefore should be under the same tag. Review Semantic

File: bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html (hunk 1 above)
Comment (jefferal1995) [reply bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html]: I create a new selector class called uppercase to resolve this format misuse.

File: bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html (hunk 2 above)
Comment (jefferal1995) [reply bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html]: I resolved it by replacing <p> by <span> and the general tag <section> of the paragraph by <p>

**Pull request merged**